subsys/logging/log_core.c

```diff
@@ -571,19 +571,20 @@ bool z_impl_log_process(void)
 	if (IS_ENABLED(CONFIG_LOG_MODE_DEFERRED)) {
 		bool dropped_pend = z_log_dropped_pending();
 		bool unordered_pend = z_log_unordered_pending();
+		uint64_t now_ms = k_uptime_get();
```

Comment:
```suggestion
		uint64_t now_ms = (uint64_t)k_uptime_get();
```
you might want to cast to unsigned type explicitly.

Reply:
done